example/fcn-xs/image_segmentaion.py

```diff
@@ -58,18 +88,35 @@ def get_data(img_path):
     return img
 
 def main():
-    fcnxs, fcnxs_args, fcnxs_auxs = mx.model.load_checkpoint(model_previx, epoch)
-    fcnxs_args["data"] = mx.nd.array(get_data(img), ctx)
+    """Module main execution"""
+    # Initialization variables - update to change your model and execution context
+    model_prefix = "FCN8s_VGG16"
+    epoch = 19
+    ctx = mx.cpu(0)
```

Comment:
Will be useful to have a commented line here for #ctx = mx.gpu()
Uncomment this line to use GPU

Reply:
Updated.